testclient/mainwindow.cpp

```diff
@@ -130,6 +131,16 @@ void MainWindow::checkPushedCommands(QByteArray bmsg)
     msg.replace(_return, QStringLiteral("\\r"));
     addSLog(msg);
 }
+
+void MainWindow::checkPrinterStatus(PrinterStatus status)
+{
+    QString msg = QString::fromLatin1("Bed:%1/%2  Extruder:%3/%4")
+                  .arg(QString::number(status.bedTemp))
+                  .arg(QString::number(status.bedTargetTemp))
+                  .arg(QString::number(status.extruderTemp))
+                  .arg(QString::number(status.extruderTargetTemp));
+    addRLog(msg);
```

Comment:
Thats true, we need to add some timers to get status of the printer in times, future PR.